docs/0001-digest-algorithms.md

```diff
@@ -0,0 +1,22 @@
+# OCFL Community Extension 0001: Digest Algorithms
+
+  * Authors: OCFL Editors
+  * OCFL Version: 1.0
+  * Obsoletes: n/a
+  * Obsoleted by: n/a
+
+## Overview
+
+This extension is an index of additional digest algorithms that are defined in community extensions. It provides a list of digest algorithm names that may be used to indicate the given algorith in `fixity` blocks of OCFL Objects, and links their defining extensions.
```

Comment:
typo: 'algorith'

Reply:
You may also wish to state that this is a 'controlled vocabulary', since there was a request for clarification about the values being used.

Reply:
Will do both :+1: